pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/signed-exchange.md

```diff
@@ -202,3 +202,10 @@ If your AMP pages were successfully distributed with Signed Exchanges, their sea
 Within the DevTools console, under the `network` tab, you will be able to see `signed-exchange` under the `type` column.  
 
 {{ image('/static/img/docs/guides/sxg/sxg3.jpg', 1366, 841, layout='responsive', alt='Within the DevTools console, under the network tab, you will be able to see signed-exchange under the type column.', caption=' ', align='' ) }}
+
+# Signed Exchange service providors 
+
+Networks that provide an AMP cache for users may generate Signed Exchanges for you. If you are using one of these networks to serve your website you will not need your own Signed Exchanges. 
```

Comment:
SXG service providers have nothing to do with AMP Caches.

Suggestion: rephrase to "Here is a list of CDNs and hosting providers offering out-of-the-box support for   Signed Exchanges. Using one of these is the easiest way to get started with SXG:"